dir.props

```diff
@@ -8,6 +8,12 @@
     <OsEnvironment Condition="'$(OsEnvironment)'==''">$(OS)</OsEnvironment>
   </PropertyGroup>
 
+  <!-- Version numbers for both managed & native binaries -->
+  <PropertyGroup>
```

Comment:
To confirm, all binaries in CoreCLR bin folder are now having file version as 4.6, right?

CC @weshaggard